src/main.rs

```diff
@@ -528,6 +528,15 @@ mod backtrace {
     }
 }
 
+fn play_scream() -> Option<()> {
```

Comment:
```suggestion
#[cfg(feature = "salty")]
fn play_scream() -> Option<()> {
```
This will fail to compile otherwise.